instagram/src/SearchBar.js

```diff
@@ -0,0 +1,27 @@
+
+import React from 'react';
+
+import './SearchBar.css';
+
+const SearchBar = (props) => {
+    return (
+       <header>
+           <div>
+            <img src="https://s3-eu-central-1.amazonaws.com/centaur-wp/designweek/prod/content/uploads/2016/05/11170038/Instagram_Logo-1002x1003.jpg" alt=""/>
```

Comment:
Good job finding these icons as images, have you looked into the font Awesome icon library? It is very compatible with React and allows you to size all icons to all be the same size.